packages/beta/package.json

```diff
@@ -18,7 +18,7 @@
     "esCheck": "es-check es5 './dist/index.js'",
     "docs": "typedoc --options typedoc.js --tsconfig tsconfig.json src/index.ts",
     "docs:bundle": "yarn docs && mkdir -p ../../docs/beta && cp -r docs/* ../../docs/beta/",
-    "extract-snippets": "rm -rf codeSnippets/ && yarn docs --json codeSnippets/docs.json && node ../../scripts/extractCodeSnippets.js",
+    "extract-snippets": "rm -rf codeSnippets/ && yarn docs --json codeSnippets/docs.json && node ../../scripts/extractCodeSnippets.js --beta",
```

Comment:
I think we don't really need to extract beta snippets (because there is no beta docs.cognite.com) yet

unless something has changed in our plans

Reply:
`test-snippets` is a part of CI. It will be also easier to move features (including code snippets) from `beta` to `stable`